### PR TITLE
Add bullet to changelog about new sorting behaviour on collections.

### PR DESCRIPTION
**Pull request by cmelchior**
Fixes https://github.com/realm/realm-java/issues/2655

@realm/java

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -19,6 +19,7 @@
 * @PrimaryKey field value can now be null for String, Byte, Short, Integer, and Long types. Older Realms should be migrated, using RealmObjectSchema.setNullable(), or by adding the @Required annotation. (#2515).
 * `RealmResults.clear()` now throws UnsupportedOperationException. Use `RealmResults.deleteAllFromRealm()` instead.
 * `RealmResults.remove(int)` now throws UnsupportedOperationException. Use `RealmResults.deleteFromRealm(int)` instead.
+* `RealmResults.sort()` and `RealmList.sort()` now returns the sorted result instead of sorting in-place.
```
Comment (zaki50) [inline CHANGELOG.md]: returns -> return ?

**Comment by zaki50** [comment]
👍  other than small comment.

**Comment by beeender** [comment]
:+1:

**Comment by kneth** [comment]
:+1: with same comment as @zaki50

**Comment by dalinaum** [comment]
:+1: besides @zaki50 comment.

**Pull request merged**